src/tests/multihost/sssd/testlib/common/utils.py

```diff
@@ -2020,36 +2020,52 @@ def __init__(self, ad_host):
 
     def get_zones(self):
         """ Returns a list of all the forward and reverse zones  on the server excluding msdcs """
-        zones = self.ad_host.run_command('dnscmd.exe /EnumZones | grep Primary | grep -v msdcs | cut -f2 -d" "')
-        return zones.stdout_text
+        zones = str.split(self.ad_host.run_command("dnscmd.exe /EnumZones").stdout_text, "\n")
+        i = 0
+        while 7 >= i:
+            del zones[0]
+            i += 1
+        i = 0
+        while 4 >= i:
+            del zones[len(zones)-1]
+            i += 1
```

Comment:
This code is quite confusing to me, what is the meaning of `7` and `4`, a comment here may help?

Reply:
It's just the junk text in the output, which is consistent.

Reply:
Python string manipulation is awful, but that's probably because I'm not good at it yet.

Reply:
Okay thank you, the comment helps.